src/bin/nydus-image/inspect.rs

```diff
@@ -50,10 +53,34 @@ impl RafsInspector {
             rafs_meta,
             bootstrap: Arc::new(Mutex::new(f)),
             cur_dir_ino: root_ino,
-            parent_inoes: Vec::new(),
+            parent_inodes: Vec::new(),
+            files_parent: BTreeMap::new(),
         })
     }
 
+    // Generate the files parent inode BTreeMap for rafs v6
+    fn generate_files_parent(&mut self) -> anyhow::Result<()> {
+        let mut files_parent = std::mem::take(&mut self.files_parent);
+        self.walk_dir(
+            self.rafs_meta.superblock.root_ino(),
+            None,
+            None,
+            &mut |parent, inode, _| {
+                if !inode.is_dir() {
+                    if let Some(parent) = parent {
```

Comment:
Is is possible that a non-dir file does not have parent?

Reply:
Maybe it is impossible. Because we walk directory from `root`.